presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/WorkerConfig.java

```diff
@@ -0,0 +1,96 @@
+package com.teradata.prestomanager.agent;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Properties;
+
+import static java.lang.String.format;
+import static java.nio.file.Files.newOutputStream;
+import static java.nio.file.Files.write;
+import static java.nio.file.StandardOpenOption.CREATE_NEW;
+import static java.util.UUID.randomUUID;
+
+public class WorkerConfig
```

Comment:
This class should be replaced with default configuration files (which should probably be in `etc/defaults`).